src/lib/sketch/About.svelte

```diff
@@ -38,13 +38,13 @@
         </ExternalLink>
```

Comment:
I'd rewrite the UX credits above to: `<li>With UX design led by <ExternalLink href="https://www.linkedin.com/in/jadene-aderonmu-b4713771">Jadene Aderonmu</ExternalLink></li>`. Very little remains today of the earlier design work, though it's been invaluable on the path to getting us here!